appveyor.yml

```diff
@@ -1,11 +1,65 @@
 image: Visual Studio 2015
 
+environment:
+  NUGET_API_KEY:
+    secure: Rh/UHYn7HGj8a7LFWo9C1rSA0sHSQNV3wIqj9bvwRR8yLmCXu9JmbNkmkctk2cBn
```

Comment:
What exactly confused you? :) 

This is the way how appveyor allows you to store sensitive information without worrying that somebody will steal it. These lines are decrypted by the build system when script runs (except for PRs). You can read more about them [here](https://www.appveyor.com/docs/build-configuration/#secure-variables).

We need to store NuGet key here to support auto-push to feeds.